asf_heat_pump_suitability/getters/load_boundaries.py

```diff
@@ -28,19 +28,15 @@ def load_gdf_local_authority_boundaries(
     elif isinstance(select_las, str):
         print(f"Loading Local Authority boundaries for {select_las}...")
         return la_boundaries_gdf[
-            la_boundaries_gdf["LAD23NM"].str.contains(select_las.title())
+            la_boundaries_gdf["LAD23NM"].str.lower() == select_las.lower()
         ]
     else:
         print(f"Loading Local Authority boundaries for {select_las}...")
-        select_las = [la.title() for la in select_las]
         la_boundaries_gdf = la_boundaries_gdf[
-            la_boundaries_gdf["LAD23NM"].str.contains("|".join(select_las))
-        ]
-        matches = [
-            la
-            for la in select_las
-            if any(la_boundaries_gdf["LAD23NM"].str.contains(la))
+            # Filter to exact LA name matches, case insensitive
+            la_boundaries_gdf["LAD23NM"].str.fullmatch("|".join(select_las), case=False)
```

Comment:
ah these changes make sense! I made some comments in the previous PR about this.